terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,36 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+# Commented params moved to vault service secrets
+# cache_server 
+# oauth2_redirect_uri 
+# oauth2_auth_uri 
+# account_url 
+log_level = "debug"
+chs_url = "https://cidev.aws.chdev.org"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "7"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chips_presenter_auth_url = "http://enablement-presenter-api:8080/presenterauth"
+chs_company_profile_api_local_url = "https://api.cidev.aws.chdev.org"
+cache_db = "0"
+dissolutions_api_url = "https://api.cidev.aws.chdev.org"
+human_log = "1"
+cookie_domain = "cidev.aws.chdev.org"
+account_local_url = "https://account.cidev.aws.chdev.org"
+pay_by_account_feature_enabled = "1"
+piwik_confirmation_page_pdf_goal_id = "998"
+piwik_landing_page_start_goal_id = "999"
+piwik_limited_company_goal_id = "999"
+piwik_partnership_goal_id = "999"
+piwik_single_director_confirmation_goal_id = "1000"
+piwik_multi_director_confirmation_goal_id = "1001"
+piwik_limited_company_confirmation_goal_id = "1002"
+piwik_partnership_confirmation_goal_id = "1003"
```

Comment:
Some of these values are also different to cidev based on the env configs here - https://github.com/companieshouse/chs-configs/blob/develop/cidev/dissolution-web/env